### ISSUE TITLE
Lunar Vision

### ISSUE THREAD

**Issue by Johan4j** [zeyad-mansour/lunar#146]
Hey! How do I make the tracking window to not be there but still track. I dont want it to even be visible to me on my monitor. I would like to make the "Lunar Vision" window to not be there.

**Comment by TonttuHomo** [comment]
Just minimize it or modify the code

**Comment by Johan4j** [comment]
> Just minimize it or modify the code

Obviously, but where do I modify the code to make the window not be there.

**Comment by TonttuHomo** [comment]
Im not sure

**Comment by TonttuHomo** [comment]
Ill look for it later

**Comment by TonttuHomo** [comment]
In the line 211, put an # before the ”cv2.imshow("Lunar Vision", frame)”. That did the trick for me!
